frontend/packages/client/src/components/Settings/SettingsSection.js

```diff
@@ -0,0 +1,55 @@
+import Blockies from 'react-blockies';
+import CopyToClipboard from 'react-copy-to-clipboard';
+import { useWebContext } from 'contexts/Web3';
+import { Svg } from '@cast/shared-components';
+import { useMediaQuery } from 'hooks';
+
+export default function SettingsSection({ walletId }) {
+  const notMobile = useMediaQuery();
+  const { injectedProvider } = useWebContext();
+
+  return (
+    <section className={'column is-flex is-flex-direction-column'}>
+      <h2 className="is-size-4 has-text-weight-bold">Settings</h2>
+      <h3 className="is-size-6 mt-2">Connected Wallet</h3>
+      <div className="is-flex is-flex-direction-row is-flex-wrap-wrap is-align-items-center">
+        <div className="mr-3 my-3">
+          <p
+            className="rounded-lg border-light flex-1 has-text-grey is-flex is-align-items-center pl-2 pt-1"
+            style={{ height: 41, width: 246 }}
+          >
+            <span>
+              <Blockies
+                seed={walletId}
+                size={notMobile ? 6.5 : 5}
+                scale={4}
+                className="blockies"
+              />
+            </span>
+            <span className="ml-1 mb-1 cursor-default">{walletId}</span>
+
+            <CopyToClipboard
+              text={walletId}
+              onCopy={() => {
+                console.log('coiped');
+              }}
```

Comment:
we should try to let the user know it worked so would be nice to have a lil state text change that says `Copied!` like I did here on vote options: https://github.com/DapperCollectives/CAST/pull/633/files#diff-2cbc0928cd1b6ff0d71f3347e651477921d2c3a7ffc840ea4bd39f6a6fb2921dR48